src/potentials.cpp

```diff
@@ -134,19 +134,21 @@ void MixerPairPotentialBase::from_json(const json &j) {
             }
         }
         if (j.count("custom") == 1) {
-            custom_pairs = j;
+            *custom_pairs = j;
         }
     } catch (const PairPotentialException &e) {
         faunus_logger->error(std::string(e.what()) + " in potential " + name);
         throw std::runtime_error("error deserialising potential " + name + " from json");
     }
+    extractorsFromJson(j);
```

Comment:
a `//` comment would be nice here

Reply:
I have added a comment locally, which I will commit later as a part of source code house-keeping.